client/package.json

```diff
@@ -11,7 +11,10 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^v5.15.4",
     "core-js": "^3.29.0",
+    "cross-spawn": "^7.0.6",
     "iframe-resizer": "^4.3.1",
+    "micromatch": "^4.0.8",
+    "nanoid": "^5.0.9",
```

Comment:
I don't think adding dependencies that weren't there before is the way to go although it does seem to resolve the `npm audit` warnings. Running `npm audit fix` from the v5 branch also fixes them without adding any new direct dependencies.